### ISSUE TITLE
The Doc for friends has a typo in the example code

### ISSUE THREAD

**Issue by fletchette** [greenheartgames/greenworks#107]
```
  greenworks.on('persona-stage-change',
                function(steam_id, persona_change_flag) {
    if (persona_change_flag == greenworks.PersonaChange.Name)
      console.log("Change to new name: " + steam_id.getPersonaName());
  });
```

Should be persona-state-change in the example.

**Comment by MadSpyxFR** [comment]
I posted a pull request #108

**Comment by hokein** [comment]
Fixed in #108.
